client/.eslintrc.js

```diff
@@ -52,6 +54,6 @@ module.exports = {
       ignoreTemplateLiterals: true,
     }],
     "no-else-return": ["error", {"allowElseIf": true}],
-    "object-curly-newline": ["error", {"consistent": true}]
+    "object-curly-newline": ["error", {"consistent": true}],
```

Comment:
While you're at it - let's turn this annoying thing off?
`"object-curly-newline": "off",`

Reply:
Actually nevermind - this is my preferred setting but for some reason didn't work in my env.